api/client/samples/crop_models/sao_paulo_sugar_content/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3594 @@
+      "text/plain": [
+       "<matplotlib.figure.Figure at 0x106c3b350>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, axes = plt.subplots(1, 1)\n",
+    "yield_df[yield_df['value']<300].plot(x='DOY', y='value', kind='scatter', c='Year', \n",
+    "                                     cmap='rainbow', ax=axes, figsize=(12,8), fontsize=20, legend=True)\n",
+    "plt.xlabel(\"Day of year\", fontsize=18)\n",
+    "plt.ylabel(\"Yield (kg/t)\", fontsize=18)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "* As we can see in this figure above, the sugar processing yield follows an increasing pattern from the Day of year (DOY) 100 to about 270 and then goes back down \n",
+    "* The processing yield between the DOY 350 and DOY 100 of the following year appear to show no pattern, and there are only data points for recent years\n",
+    "* For the purpose of this analysis, we therefore limit the study period to between the DOY 100 and 350\n",
+    "* **Note**: It is possible to limit the time period of the data being pulled. You just need to specify 'start_date' and 
```

Comment:
Why is this 0.9218 value hardcoded? Why not use a variable?

Reply:
changed into a variable